.readme-partials.yaml

```diff
@@ -0,0 +1,101 @@
+introduction: |-
+  > This module provides Cloud Debugger support for Node.js applications.
+  Cloud Debugger is a feature of Google Cloud Platform that lets you debug your
+  applications in production without stopping or pausing your application.
+
+body: |-
+  ## Debugger Agent Settings
+
+  You can customize the behaviour of the automatic debugger agent. See
+  [the agent configuration][config-ts] for a list of possible configuration
+  options. These options can be passed in the `options` object passed to the
+  `start` function.
+
+  ```js
+  require('@google-cloud/debug-agent').start({
+    // .. auth settings ..
+
+    // debug agent settings:
+    allowExpressions: true,
+    serviceContext: {
+      service: 'my-service',
+      version: 'version-1'
+    },
+    capture: { maxFrames: 20, maxProperties: 100 }
+  });
+  ```
+
+  ## Using the Debugger
+
+  Once your application is running (deployed, or elsewhere), you should be able
```

Comment:
what do you mean by 'elsewhere'? I would either define or remove the content in ()

Reply:
remove "should be able" and rephrase as a more concrete "once your application is running, you can use the Debug UI in your Cloud Developer console.